luma/lcd/device.py

```diff
@@ -292,6 +292,95 @@ def contrast(self, value):
         self.command(0x21, 0x14, value | 0x80, 0x20)
 
 
+class st7789(backlit_device):
+    """
+    Serial interface to a colour ST7789 240x240 pixel LCD display.
+    """
+    def __init__(self, serial_interface=None, rotate=0, **kwargs):
+        super(st7789, self).__init__(luma.lcd.const.st7789, serial_interface, **kwargs)
+        self.capabilities(240, 240, rotate, mode="RGB")
+
+        self.command(0x36)
```

Comment:
I think it may be possible to combine:

```python
self.command(0x36)
self.data([0x70])
```

into:

```python
self.command(0x36, 0x70)
```

have a look at the implementation for `command` for more details

Reply:
Also it would be quite nice to document each command, eg like: https://github.com/rm-hull/luma.lcd/blob/master/luma/lcd/device.py#L460-L482

Also here you can see the combined command/data

Reply:
Yeah, I like this combined syntax more, I wonder why author of the original PR went that way.